### DISCUSSION TITLE
Motorola ZN5 (MotoMAGX LJ6.3)

### DISCUSSION THREAD

**Discussion by KathyTi** [mahomaps/mm-v1#35] [Tested devices] [answered]
Непосредственно перед стартом на момент появляется предупреждение(видно окно с восклицательным знаком) и программа виснет на сплеш скрине. Скорее всего хипа(до 4мб) не хватает.

**Comment by Feodor0090** [comment]
Хипа должно хватать. Предупреждение может быть только системное, скорее всего про неподдерживаемые JSR. Будем разбираться.

**Reply by KathyTi** [reply]
Вижу только окно предупреждения. Текста нет, всё происходит быстро и вот уже сплеш.

**Comment by Feodor0090** [comment]
Бампнуто до #40

**Reply by KathyTi** [reply, marked as answer]
Благодаря быстрой ОС разработчиков всё заработало.
![1685206466_030234](https://github.com/mahomaps/mm-v1/assets/63660188/b1124278-5edc-4985-8a5b-d905902bcd8c)
![1685206554_030239](https://github.com/mahomaps/mm-v1/assets/63660188/03329d8d-6e44-4113-ac93-09286549b0e6)
![1685206561_030240](https://github.com/mahomaps/mm-v1/assets/63660188/86a9a7f7-9d9c-467b-a1bb-152cba2cbed0)
![1685206578_030242](https://github.com/mahomaps/mm-v1/assets/63660188/3011ba59-7fde-417d-b60d-ddcb9e881a3e)
![1685206636_030245](https://github.com/mahomaps/mm-v1/assets/63660188/afb42c86-b29b-4350-a20e-b473beef76b3)
![1685206748_030247](https://github.com/mahomaps/mm-v1/assets/63660188/012288ae-e1e6-47b0-8602-a9a11d3e5fe2)
![1685206727_030246](https://github.com/mahomaps/mm-v1/assets/63660188/47d7dc3b-1567-4de6-b3d9-a3c2b4ca3ebb)
![1685206338_030230](https://github.com/mahomaps/mm-v1/assets/63660188/11942ad5-0b7c-4f79-a70c-f6e62a07d609)
![1685206357_030231](https://github.com/mahomaps/mm-v1/assets/63660188/b17a0f36-6ac7-4fc8-9d26-b33208b9f740)
![Uploading 1685206455_030233.png…]()
